package.json

```diff
@@ -32,8 +35,8 @@
     ]
   },
   "dependencies": {
-    "@types/markdown-it": "10.0.2",
-    "markdown-it": "11.0.0"
+    "@types/markdown-it": "^10.0.2",
+    "markdown-it": "^11.0.0"
```

Comment:
Why did you readd the version ranges?

Reply:
It's not a good idea to have fixed versions in npm packages as it forces npm to install two different versions of the same package if multiple packages have common dependencies.

This is not only a waste of space but also causes builds to break as soon as typescript is involved.

Reply:
When we installed renovate bot we read https://docs.renovatebot.com/dependency-pinning/#pinning-dependencies-and-lock-files which explains when you should pin your dependencies and when you should not.
TL;DR: If you are a dependency, then you should keep semver ranges. This is a dependency, so it's okay for me.